changelog.md

```diff
@@ -6,6 +6,43 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### 0.41.0
+
+### Added
+
+* `state init` is now a stable command, meaning you no longer need to opt-in to
+  unstable commands to use it.
+* Signing up for a new account now opens the account creation page in your
+  browser. Instead of bringing you to the sign in page.
+* `state shell` can now detect currently active subshells preventing nested
+  shells from being created.
+* The State Tool will now print a message when starting a subshell, indicating
+  the project that is being used.
+* Wildcard and partial version matching is now supported for `state install` 
+  and for language versions with `state init`.
+  For example: 
```

Comment:
Do you intend for this to be rendered on a new line? It's not. Same for the one on line 29.

Reply:
Whoops, yes I did. Thanks for catching that.

Reply:
Though it looks from your suggestion that you'd prefer it be on one line so I've made that change. Also removed the second example with the wildcards.

Reply:
Oh I didn't mean to suggest that change, I was trying to figure out what this should look like an accidentally submitted it.